tests/qa_automation/kernel_kexec.pm

```diff
@@ -46,7 +46,7 @@ sub run {
     # don't use built-in systemctl api, see poo#31180
     script_run("systemctl kexec", 0);
     # wait for reboot
-    reset_consoles();
+    $self->wait_boot;
```

Comment:
won't work because we would be stuck waiting for grub2 when the system already booted to text login prompt. I am handling it